azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/implementation/AvailabilitySetsImpl.java

```diff
@@ -96,7 +99,7 @@ public void delete(String groupName, String name) throws Exception {
         this.client.delete(groupName, name);
     }
 
-    /** Fluent model create helpers **/
+    /** Fluent model create helpers. **/
```

Comment:
actually, this is not intended to be a Javadoc. I replaced my use of similar comments with //, so that checkstyle wouldn't mistaken these as javadocs.